### ISSUE TITLE
Allow to run the migration manually

### ISSUE THREAD

**Issue by loicmathieu** [micronaut-projects/micronaut-flyway#475]
### Feature description

Currently, if we disable Flyway migration, there is no way to inject a Flyway object and run the migration by ourselves.

For some edge cases, we need a way to control the Flyway migration, so we need a way to access the Flyway object and run the migration by ourselves.

**Comment by jeremyg484** [comment]
@loicmathieu Does the method described [here](https://micronaut-projects.github.io/micronaut-flyway/latest/guide/#jpa) under "Run migrations manually" not work for these edge cases? If not, what is missing that you need?

**Comment by loicmathieu** [comment]
@jeremyg484 yes it fit our needs, it seems to be new or I miss it when I read the doc.
